guides/integrate-wordpress.md

```diff
@@ -128,11 +127,11 @@ Congratulations, you have configured the BI console and the OIDC client plugin.
 
 ### Try logging in
 
-  We will now attempt to log in and verify successful authentication.
+We will now attempt to log in and verify successful authentication.
 
-  1. Using a device where you have [created an identity and then enrolled a credential](#Create an Identity in the BI console), Visit http://your_hostname/wp-login.php to login
-  1. You will see a "Login with OpenID Connect" button. Click it.
-  1. You will be redirected to the Beyond Identity Web Authenticator. 
-  1. You may see a step-up authentication prompt, depending on how Policy is set up for your tenant.
-  1. At the conclusion of a successful authentication, you will be redirected to your wordpress console, and
-  1. You will see several successful authentication events in BI Admin Console -> Events
+1. Using a device where you have [created an identity and then enrolled a credential](#Create an Identity in the BI console), Visit http://your_hostname/wp-login.php to login
+1. You will see a "Login with OpenID Connect" button. Click it.
```

Comment:
Does it always show this button? The current came web app is using Next Auth but we don't see this button and you don't need to click it to authenticate with beyond identity. Is there a way to configure the provider to just go ahead and authenticate?

Reply:
Hmm, you may be right. In their example app, I didn't create a button, but they may be generating them based on the providers. I'll rephrase this.

Reply:
When I was playing around with it in the acme web app it seems like it defaulted to a button to click if I didn't have something set up correctly in the auth provider. This is what we're using in Acme:

https://github.com/gobeyondidentity/acme/blob/main/web/src/pages/api/auth/%5B...nextauth%5D.ts